### ISSUE TITLE
Translators // Translations

### ISSUE THREAD

**Issue by RichardLitt** [nodeschool/chiang-mai#4]
Does anyone know what the state of the Node workshoppers is in Thai?

**Comment by rburns** [comment]
My guess would be not so good. A quick glance at a couple random workshoppers https://github.com/sethvincent/javascripting/tree/master/i18n https://github.com/workshopper/learnyounode/tree/master/i18n doesn't turn up anything. Thai also appears to be missing from the website localizations http://nodeschool.io/

It's maybe more effective for this first time event to make note of mentors and/or participants who can help out with translation as necessary.

**Comment by RichardLitt** [comment]
Fair. In Bali, we had a translator attend who could help out, organized through the Workspace. I don't know how many Thai coders are coming, at the moment (or even how to reach them, really).

**Comment by rburns** [comment]
less of a guess, though possibly not entirely accurate. (https://gist.github.com/rburns/e4d1dc964b1c8a6fdae9)

stevekane/promise-it-wont-hurt -- en, fr
azat-co/expressworks -- en, fr
timoxley/functional-javascript-workshop -- en, fr
npm/how-to-npm -- none
bulkan/async-you -- en, fr
sporto/planetproto -- en, fr
stackgl/webgl-workshop -- none
tableflip/nodebot-workshop -- none
othiym23/bug-clinic -- none
alexmackey/IntroToWebGLWithThreeJS -- none
koajs/kick-off-koa -- en, fr
domenic/count-to-6 -- none
substack/browserify-adventure -- none
mdunisch/lololodash -- en, fr
robertkowalski/learnyoucouchdb -- none
thlorenz/learnuv -- none
isRuslan/learn-generators -- en, fr
jlord/git-it -- none
substack/stream-adventure -- none
sethvincent/javascripting -- es, ja, ko
rvagg/levelmeup -- none
maxogden/bytewiser -- none
spumko/makemehapi -- en, fr
gl-modules/shader-school -- none
rvagg/learnyounode -- en, es, fr, ja, pt-br, ru, zh-cn, zh-tw
rvagg/goingnative -- none
finnp/test-anything -- none

Looks to be quite an opportunity for doing localizations.

I would expect Thai's that do show up to either not have any problem with English workshoppers, or not have a problem finding a mentor or other participant with the capacity to help out. I can volunteer for the latter case.  I can also help in trying to reach some local folks.

**Comment by RichardLitt** [comment]
Yeah; that's a pretty small list of translations. I would also expect the same, especially from coders. If you could try to reach some local folks, that would be great, but I suspect we might just need a pairing system to help out. 

Maybe we could try and translate learnyounode into Thai, that's the most commonly done module.
